app/subscriber/src/components/basic-search/BasicSearch.tsx

```diff
@@ -45,8 +45,8 @@ export const BasicSearch = ({ onSearch, inHeader }: IBasicSearchProps) => {
 
   return (
     <styled.BasicSearch inHeader={inHeader}>
-      <label>SEARCH FOR: </label>
       <Row className="search-row">
+        <label>SEARCH FOR: </label>
```

Comment:
If we're doing CSS updates, could you change any instances of ALL caps in these files you are touching to mixed case and then apply a CSS `text-transform:uppercase` to them please?